tests/js/common/replication_api/shell-api-replication-global.js

```diff
@@ -0,0 +1,1183 @@
+      let all = doc.parsedBody;
+      assertTrue(all.hasOwnProperty('state'));
+      assertTrue(all.hasOwnProperty('server'));
+
+      let state = all['state'];
+      assertFalse(state['running']);
+      assertTrue(state.hasOwnProperty("lastAppliedContinuousTick"));
+      assertTrue(state.hasOwnProperty("lastProcessedContinuousTick"));
+      assertTrue(state.hasOwnProperty("lastAvailableContinuousTick"));
+      assertTrue(state.hasOwnProperty("safeResumeTick"));
+
+      assertTrue(state.hasOwnProperty("progress"));
+      let progress = state['progress'];
+      assertTrue(progress.hasOwnProperty("time"));
+      assertMatch(/^([0-9]+-[0-9]+-[0-9]+T[0-9]+:[0-9]+:[0-9]+Z)?$/, progress['time']);
+      assertTrue(progress.hasOwnProperty("failedConnects"));
+
+      assertTrue(state.hasOwnProperty("totalRequests"));
+      assertTrue(state.hasOwnProperty("totalFailedConnects"));
+      assertTrue(state.hasOwnProperty("totalEvents"));
+      assertTrue(state.hasOwnProperty("totalOperationsExcluded"));
+
+      assertTrue(state.hasOwnProperty("lastError"));
+      let lastError = state["lastError"];
+      assertTrue(lastError.hasOwnProperty("errorNum"));
+      assertTrue(state.hasOwnProperty("time"));
+      assertMatch(/^[0-9]+-[0-9]+-[0-9]+T[0-9]+:[0-9]+:[0-9]+Z$/, state['time']);
+    }
+  };
+}
+
+////////////////////////////////////////////////////////////////////////////////;
+// wal access;
+////////////////////////////////////////////////////////////////////////////////;
+function dealing_with_wal_access_apSuite () {
+  let api = "/_api/wal";
+  return {
+
+    setUp: function() {
+      db._drop("UnitTestsReplication");
+    },
+
+    tearDown: function() {
+      db._drop("UnitTestsReplication");
+    },
+
+    ////////////////////////////////////////////////////////////////////////////////;
+    // state;
+    ////////////////////////////////////////////////////////////////////////////////;
+
+    test_check_the_state: function() {
+      // fetch state;
+      let cmd = "/_api/replication/logger-state";
+      let doc = arango.GET_RAW(cmd);
+
+      assertEqual(doc.code, 200);
+
+      let all = doc.parsedBody;
+      assertTrue(all.hasOwnProperty('state'));
+      assertTrue(all.hasOwnProperty('server'));
+      assertTrue(all.hasOwnProperty('clients'));
+
+      let state = all['state'];
+      assertTrue(state['running']);
+      assertMatch(/^[0-9]+$/, state['lastLogTick']);
+      assertMatch(/^[0-9]+-[0-9]+-[0-9]+T[0-9]+:[0-9]+:[0-9]+Z$/, state['time']);
+
+      let server = all['server'];
+      assertMatch(/^[0-9]+$/, server['serverId']);
+      assertTrue(server.hasOwnProperty('version'));
+    },
+
+
+    ////////////////////////////////////////////////////////////////////////////////;
+    // tickRanges;
+    ////////////////////////////////////////////////////////////////////////////////;
+
+    test_fetches_the_available_tick_range_1: function() {
+      // fetch state;
+      let cmd = api + "/range";
```

Comment:
As far as I can see, this test has nothing to do with the "global" applier.
Probably all tests that are not specific to the global applier should be moved into a separate test file.